kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/IcebergWriterCompatV1Suite.scala

```diff
@@ -34,7 +34,13 @@ import io.delta.kernel.utils.CloseableIterable.emptyIterable
 import org.assertj.core.api.Assertions.assertThat
 import org.scalatest.funsuite.AnyFunSuite
 
-class IcebergWriterCompatV1Suite extends AnyFunSuite with WriteUtils
+class IcebergWriterCompatV1TransactionBuilderV1Suite extends IcebergWriterCompatV1SuiteBase
```

Comment:
Oop -- this will conflict with https://github.com/delta-io/delta/pull/5176/files#diff-e0070a88a6adfae74e632cb5052717e1a422de8bc5f33646ef0230b78ed66553. My PR appears to be a superset of this file.

I'm happy to either (a) one of us merges first and we just resolve conflicts later, or (b) you revert these changes